### ISSUE TITLE
pack install fails with git > 1.9.0 

### ISSUE THREAD

**Issue by lakshmi-kannan** [StackStorm/st2#3477]
```
(virtualenv)vagrant@st2dev /o/s/p/vsphere ❯❯❯ st2 pack install vyatta                                                              master ✱

        [  failed   ] download pack

id: 59417552d9d7ed69b9b2d9c3
action.ref: packs.install
parameters:
  packs:
  - vyatta
status: failed
error: Traceback (most recent call last):
  File "/mnt/src/storm/st2/st2common/st2common/runners/python_action_wrapper.py", line 259, in <module>
    obj.run()
  File "/mnt/src/storm/st2/st2common/st2common/runners/python_action_wrapper.py", line 155, in run
    output = action.run(**self._parameters)
  File "/opt/stackstorm/packs/packs/actions/pack_mgmt/download.py", line 75, in run
    verifyssl=verifyssl, ref=pack_version)
  File "/opt/stackstorm/packs/packs/actions/pack_mgmt/download.py", line 150, in _clone_repo
    repo.git.checkout('-b', short_branch, branch)
  File "/mnt/src/storm/st2/virtualenv/lib/python2.7/site-packages/git/cmd.py", line 425, in <lambda>
    return lambda *args, **kwargs: self._call_process(name, *args, **kwargs)
  File "/mnt/src/storm/st2/virtualenv/lib/python2.7/site-packages/git/cmd.py", line 877, in _call_process
    return self.execute(call, **exec_kwargs)
  File "/mnt/src/storm/st2/virtualenv/lib/python2.7/site-packages/git/cmd.py", line 688, in execute
    raise GitCommandError(command, status, stderr_value, stdout_value)
git.exc.GitCommandError: Cmd('git') failed due to: exit code(128)
  cmdline: git checkout -b master master
  stderr: 'fatal: Cannot update paths and switch to branch '?[7;33mmaster?[m' at the same time.
Did you intend to checkout '?[7;33mmaster?[m' which can not be resolved as commit?'

traceback: None
failed_on: download pack
start_timestamp: 2017-06-14T17:41:38.485470Z
end_timestamp: 2017-06-14T17:41:43.411575Z
+--------------------------+---------------------+---------------+----------------+----------------------+
| id                       | status              | task          | action         | start_timestamp      |
+--------------------------+---------------------+---------------+----------------+----------------------+
| 59417553d9d7ed6975cd0615 | failed (4s elapsed) | download pack | packs.download | Wed, 14 Jun 2017     |
|                          |                     |               |                | 17:41:39 UTC         |
+--------------------------+---------------------+---------------+----------------+----------------------+
(virtualenv)vagrant@st2dev /o/s/p/vsphere ❯❯❯
```

**Comment by Kami** [comment]
I wonder if that's specific to your environment - e.g. shell or similar.

It works fine for me using `git version 1.9.1`.

**Comment by LindsayHill** [comment]
is this still an issue, or can it be closed? Can't reproduce on my system with 1.9.1

**Comment by LindsayHill** [comment]
No response from user, closing issue.
